### DISCUSSION TITLE
More entitiys like from the RCT-Portal

### DISCUSSION THREAD

**Discussion by seemedFilm** [weltenwort/home-assistant-rct-power-integration#350] [Q&A]
I would be very happy if it would be possible to get entitys as they are provided by the RCT-Power Portal.
I could already identify two by myself, it would be awesome if there is a chance to have for each showed value an own entity.
[https://imgur.com/a/6nMuRs2](url)

In the linked picture, I've put numbers below the entitys which I talk about.

1:
2:
3:
4:
5: sensor.rct_power_external_energy_production_day
6:
7: sensor.rct_power_grid_energy_production_day

Switched from "Today" to "All"
8: sensor.rct_power_all_generators_energy_production_total
9:
10: sensor.rct_power_grid_energy_production_absolute_total

**Comment by weltenwort** [comment]
Hey @Sectorchan, sorry for the delayed answer. I've never used the portal, but based on your screenshot this is my interpretation:

1. `sensor.rct_power_all_generators_power`
2. `sensor.rct_power_consumer_power`
3. `sensor.rct_power_battery_power`
4. `sensor.rct_power_grid_power`
6. This is not provided directly by the inverter, but calculated. If you use the energy dashboard of Home Assistant it will provide a percentage value for this.
9. same
